lint.go

```diff
@@ -52,12 +52,12 @@ func (p *Problem) String() string {
 	return p.Text
 }
 
-type byPosition []Problem
+type ByPosition []Problem
```

Comment:
this would need a different name if it is to be exported. What's the use case for this though?